gateway/1.8/modules/ROOT/nav.adoc

```diff
@@ -95,7 +95,7 @@
 *** xref:flex-view-api-metrics.adoc[View Key Metrics for Flex Gateway APIs]
 *** xref:flex-view-replica-status.adoc[View the Replica Status in Runtime Manager]
 *** xref:flex-view-api-status.adoc[View the API Status in Runtime Manager]
-*** xref:flex-view-logs-in-monitoring.adoc[View Logs in Anypoint Monitoring]
+*** xref:flex-view-logs-in-runtime-manager.adoc[]
```

Comment:
```suggestion
*** xref:flex-view-logs-in-runtime-manager.adoc[View Logs in Runtime Manager]
```
Consistency with the other imperative page titles. Leaving empty brackets will result in a title with a gerund. This requirement goes back to the days of Kim. If we want all our page titles to contain gerunds instead, we'll need to change most if not all nav page titles.